src/Toolkit/Toolkit/UI/Controls/FeatureForm/FeatureFormView.cs

```diff
@@ -357,6 +358,28 @@ internal bool OnFormAttachmentClicked(FormAttachment attachment)
             }
             return false;
         }
+
+        /// <summary>
+        /// Raised when the Barcode Icon in a barcode element is clicked.
+        /// </summary>
+        /// <remarks>
+        /// <para>Set the <see cref="BarcodeButtonClickedEventArgs.Handled"/> property to <c>true</c> to prevent
+        /// any default code and perform your own logic.</para>
+        /// </remarks>
+        public event EventHandler<BarcodeButtonClickedEventArgs>? BarcodeButtonClicked;
+
+        internal bool OnBarcodeButtonClicked(FieldFormElement element)
```

Comment:
Should this button be clickable (or even visible) when the field is read-only?

Reply:
Nope and it isn't: 

https://github.com/Esri/arcgis-maps-sdk-dotnet-toolkit/blob/c46755d8ab57f4dff03512e0b12ed6ce11da1fa0/src/Toolkit/Toolkit/UI/Controls/FeatureForm/TextFormInputView.cs#L95
https://github.com/Esri/arcgis-maps-sdk-dotnet-toolkit/blob/c46755d8ab57f4dff03512e0b12ed6ce11da1fa0/src/Toolkit/Toolkit/UI/Controls/FeatureForm/TextFormInputView.Maui.cs#L71
https://github.com/Esri/arcgis-maps-sdk-dotnet-toolkit/blob/c46755d8ab57f4dff03512e0b12ed6ce11da1fa0/src/Toolkit/Toolkit.WPF/UI/Controls/FeatureForm/FeatureFormView.Theme.xaml#L178-L180